validator_client/src/attestation_service.rs

```diff
@@ -377,12 +377,16 @@ impl<T: SlotClock + 'static, E: EthSpec> AttestationService<T, E> {
                 signature: AggregateSignature::infinity(),
             };
 
-            if let Err(e) = self.validator_store.sign_attestation(
-                duty.pubkey,
-                duty.validator_committee_index as usize,
-                &mut attestation,
-                current_epoch,
-            ) {
+            if let Err(e) = self
+                .validator_store
+                .sign_attestation(
+                    duty.pubkey,
+                    duty.validator_committee_index as usize,
+                    &mut attestation,
+                    current_epoch,
+                )
+                .await
```

Comment:
We should probably consider parallelising here too, but it's less urgent because we're unlikely to have multiple attesters for the same committee/index except on small networks.

Reply:
There's also an issue whereby the slashing protection database is going to block some amount of parallelism, by requiring that all the slashing protection checks succeed before signing requests are made, see: https://github.com/sigp/lighthouse/issues/1914#issuecomment-917934011